src/components/appTile/AppMenu.js

```diff
@@ -0,0 +1,146 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import { IconButton, makeStyles, Menu, MenuItem } from "@material-ui/core";
+import MoreVertIcon from "@material-ui/icons/MoreVert";
+import withI18N, { formatMessage, getMessage } from "../../util/I18NWrapper";
+import { injectIntl } from "react-intl";
+import InfoIcon from "@material-ui/icons/InfoOutlined";
+import UnFavoriteIcon from "@material-ui/icons/FavoriteBorderOutlined";
+import FavoriteIcon from "@material-ui/icons/Favorite";
+import CommentsIcon from "@material-ui/icons/CommentOutlined";
+import palette from "../../util/CyVersePalette";
+import intlData from "./messages";
+import build from "../../util/DebugIDUtil";
+import ids from "./ids";
+
+/**
+ *
+ * @author sriram
+ *
+ * Menu items for Apps Tile and table view
+ *
+ */
+const useStyles = makeStyles((theme) => ({
+    menuItem: {
+        padding: 5,
+    },
+    toolbarItemColor: {
+        color: palette.darkBlue,
+    },
+}));
+
+function AppMenu(props) {
+    const {
+        onAppInfoClick,
+        onCommentsClick,
+        onFavoriteClick,
+        baseDebugId,
+        isExternal,
+        isFavorite,
+        intl,
+    } = props;
+    const [anchorEl, setAnchorEl] = useState(null);
+    const open = Boolean(anchorEl);
+    const handleClose = () => setAnchorEl(null);
+    const classes = useStyles();
+    return (
+        <div>
+            <IconButton
+                id={build(baseDebugId, ids.DOT_MENU)}
+                aria-label={formatMessage(intl, "more")}
+                aria-owns={open ? "long-menu" : null}
+                aria-haspopup="true"
+                onClick={(event) => setAnchorEl(event.currentTarget)}
+                size="small"
+            >
+                <MoreVertIcon />
+            </IconButton>
+            <Menu anchorEl={anchorEl} open={open} onClose={handleClose}>
+                <MenuItem
+                    id={build(baseDebugId, ids.APP_INFO)}
+                    className={classes.menuItem}
+                    data-disabled={false}
```

Comment:
Are we intentionally keeping `data-disabled={false}` for QA tests?

Reply:
Thats the idea. You can write a test to make sure if the menuItem is enabled or disabled conditions.